.github/workflows/build-and-test.yml

```diff
@@ -24,7 +24,7 @@ jobs:
       uses: actions/setup-java@b36c23c0d998641eff861008f374ee103c25ac73 # v4
       with:
         distribution: temurin
-        java-version: 17
+        java-version: 21
```

Comment:
What about `java-version: '11'` in `.github/workflows/docker-build.yml`?

Reply:
I also switched this to 21. I was not sure whether there is a specific reason why here a different Java version should be used. Maybe an oversight?

Reply:
I guess so.